examples/Simple_Example.ipynb

```diff
@@ -29,7 +29,7 @@
   {
    "cell_type": "code",
    "source": [
-    "!pip install git+https://github.com/apache/mahout.git@main"
+    "# pip install git+https://github.com/apache/mahout.git@main"
```

Comment:
Is this supposed to be commented out?

Reply:
Yes, for notebooks to actually be tested on the code that is being checked in it does.